src/client/theme-default/components/NavLinks.vue

```diff
@@ -9,7 +9,7 @@ import NavDropdownLink from './NavDropdownLink.vue'
 const { theme } = useData()
 const localeLinks = useLocaleLinks()
 const repo = useRepo()
-const show = computed(() => theme.value.value || repo.value)
+const show = computed(() => theme.value.nav || repo.value || localeLinks)
```

Comment:
It needs to read the `.value`, otherwise this would always be true

```suggestion
const show = computed(() => theme.value.nav || repo.value || localeLinks.value)
```

Reply:
Damned, you're right! Thanks for the point!